docs/zh/UserGuide/Server/Cluster Setup.md

```diff
@@ -20,5 +20,235 @@
 -->
 
 # 集群设置
+安装环境请参考[安装环境](../Get%20Started/QuickStart.md)
+## 集群环境搭建
+用户可以搭建伪分布式模式或是分布式模式，伪分布式模式和分布式模式的主要区别是配置文件中seed_nodes的不同，配置项含义请参考[配置项](#配置项)。
+启动其中一个节点的服务，需要执行如下命令：
 
-Coming Soon.
\ No newline at end of file
+```bash
+# Unix/OS X
+> nohup sbin/start-node.sh >/dev/null 2>&1 &
+or
+> nohup sbin/start-node.sh -c <conf_path> -internal_meta_port 9003 >/dev/null 2>&1 &
+
+# Windows
+> nohup sbin/start-node.bat
+or
+> nohup sbin/start-node.bat -c <conf_path> -internal_meta_port 9003
+or
+```
+
+-c <conf_path>使用conf_path文件夹里面的配置文件覆盖默认配置文件; -internal_meta_port 9003覆盖特定配置项internal_meta_port的配置，
+目前支持的启动覆盖原有配置的配置项有：
+internal_meta_port、internal_data_port、cluster_rpc_port、seed_nodes。当两者都存在的时候，指定配置项的配置会覆盖配置文件中的配置。
+
+## 配置项
+
+为方便IoTDB Server的配置与管理，IoTDB Server为用户提供三种配置项，使得用户可以在启动服务器或服务器运行时对其进行配置。
+
+三种配置项的配置文件均位于IoTDB安装目录：`$IOTDB_HOME/conf`文件夹下,其中涉及server配置的共有2个文件，分别为：`iotdb-cluster.properties`、`iotdb-engine.properties`、`iotdb-env.sh`, 用户可以通过更改其中的配置项对系统运行的相关配置项进行配置。
+
+配置文件的说明如下：
+
+* `iotdb-env.sh`：环境配置项的默认配置文件。用户可以在文件中配置JAVA-JVM的相关系统配置项。
+
+* `iotdb-engine.properties`：IoTDB引擎层系统配置项的默认配置文件。用户可以在文件中配置IoTDB引擎运行时的相关参数，如JDBC服务监听端口(`rpc_port`)、overflow数据文件存储目录(`overflow_data_dir`)等。此外，用户可以在文件中配置IoTDB存储时TsFile文件的相关信息，如每次将内存中的数据写入到磁盘时的数据大小(`group_size_in_byte`)，内存中每个列打一次包的大小(`page_size_in_byte`)等。
+
+* `iotdb-cluster.properties`: IoTDB集群所需要的一些配置。
+
+`iotdb-engine.properties`、`iotdb-env.sh` 两个配置文件详细说明请参考[配置手册](Config%20Manual.md)，下面描述的配置项是在`iotdb-cluster.properties`文件中的。
+
+* internal\_meta\_port
+
+|名字|internal\_meta\_port|
+|:---:|:---|
+|描述|IoTDB meta服务端口，**IoTDB将为每个meta服务自动创建心跳端口。默认meta服务心跳端口为internal_meta_port+1，请确认这两个端口不是系统保留端口并且未被占用**|
+|类型|Int32|
+|默认值|9003|
+|改后生效方式|重启服务器生效|
+
+* internal\_data\_port
+
+|名字|internal\_data\_port|
+|:---:|:---|
+|描述|IoTDB data服务端口，**IoTDB将为每个data服务自动创建心跳端口。默认的data服务心跳端口为internal_data_port+1。请确认这两个端口不是系统保留端口并且未被占用**|
+|类型|Int32|
+|默认值|40010|
+|改后生效方式|重启服务器生效|
+
+* cluster\_rpc\_port
+
+|名字|cluster\_rpc\_port|
+|:---:|:---|
+|描述|与客户端通信的端口，请确认该端口不是系统保留端口并且未被占用|
+|类型|Int32|
+|默认值|55560|
+|改后生效方式|重启服务器生效|
+
+* seed\_nodes
+
+|名字|seed\_nodes|
+|:---:|:---|
+|描述|集群中最初节点的地址，{IP/DOMAIN}:internal\_meta\_port:internal\_data\_port格式，用逗号分割；对于伪分布式模式，可以都填写localhost，或是127.0.0.1 或是混合填写，但是不能够出现真实的ip地址；对于分布式模式，支持填写real ip 或是hostname，但是不能够出现localhost或是127.0.0.1|
+|类型|String|
+|默认值|seed_nodes=127.0.0.1:9003:40010,127.0.0.1:9005:40012,127.0.0.1:9007:40014|
+|改后生效方式|重启服务器生效|
+
+* rpc\_thrift\_compression\_enable
+
+|名字|rpc\_thrift\_compression\_enable|
+|:---:|:---|
+|描述|是否开启thrift压缩通信，**注意这个参数要各个节点保持一致**|
```

Comment:
I think it's better to point out that this parameter only applies to internal communications in this doc to avoid possible confusion. If user want to change compression settings for external clients, they should modify 'rpc_thrift_compression_enable' in 'iotdb-engine.properties'.

Reply:
Thanks for all your advice！I will fix it